components/phpunit_bridge.rst

```diff
@@ -624,8 +624,9 @@ its ``bin/simple-phpunit`` command. It has the following features:
 
 * Does not embed ``symfony/yaml`` nor ``prophecy`` to prevent any conflicts with
   these dependencies;
-* Uses PHPUnit 4.8 when run with PHP <=5.5, PHPUnit 5.7 when run with PHP >=5.6
-  and PHPUnit 6.5 when run with PHP >=7.2;
+* Uses PHPUnit 4.8 when run with PHP <=5.5, PHPUnit 5.7 when run with PHP 5.6,
```

Comment:
im not sure this is a feature really :) i'd prefer a "compatible version" table and keep it straightforward (as this quite a sentence)